### PR TITLE
Allaway/round robin

### PR DESCRIPTION
**Pull request by allaway**
### Purpose

Start of #88, but not complete. When reading the intro and the sections, it seemed like they were almost in reverse order (methods for analysis, methods for dealing with paucity of data, methods for characterizing data). I flipped these, so now it's methods for characterizing data, methods for dealing with paucity of data, methods for analysis.

I didn't want to make such a drastic change and then also combine with edits, so once this is merged or rejected i'll get back to my round robin. 

### Directions for reviewers

look at the new order and see if it makes sense

#### Is there anything that you want to discuss further?

Does this reorg make sense? I also consolidated all of the methods into one section.

#### Is the pull request ready for review?

Yes

### Manuscript checklist

Unless otherwise noted above, this PR will be considered ready for review when all four items have been checked.

- [x] All changes to text follow "one sentence per line" [[Manubot instructions](https://github.com/jaybee84/ml-in-rd/blob/draft-branch/USAGE.md#manubot-markdown)]
- [x] All citations follow the [Manubot citation instructions](https://github.com/jaybee84/ml-in-rd/blob/draft-branch/USAGE.md#citations)
- [x] All changes or additions to tables follow the [Manubot table instructions](https://github.com/jaybee84/ml-in-rd/blob/draft-branch/USAGE.md#tables)
- [x] All figures follow the [Manubot figure instructions](https://github.com/jaybee84/ml-in-rd/blob/draft-branch/USAGE.md#figures)
- [x] All terms that need to be added to glossary have been added to #50 
- [x] There are no new spelling errors identified by the Manubot spellchecker

### CODE REVIEW

**Comment by allaway** [comment]
Argh, I screwed up the order...

**Comment by allaway** [comment]
OK, fingers crossed I did it right this time!

**Review by jaclyn-taroni — APPROVED** [review]
New order seems reasonable to me 👍

File: content/02.intro.md [hunk 1]
```diff
@@ -1,16 +1,13 @@
...
+
+Rare disease research has additional constraints to consider when using machine learning methods, including lack of statistical power in dataset size, heterogeneity in available data, and sensitivity of machine learning methods to misinterpretation in view of small datasets. 
+Moreover, in the context of rare disease, special considerations need to be made to safeguard against misinterpretation of results.
 Such considerations include incorporation of techniques that build upon prior domain-specific knowledge, methods that are resilient to challenges posed by small datasets, and methods that can mitigate technical disparities in the data.
+Recent advances in methodologies to accommodate rarity of samples and increased transparency in model outputs have encouraged application of machine learning in rare diseases.
+In this perspective, we discuss techniques for understanding the nature of rare disease data data, methods for addressing some of the limitations of these data, and machine learning methods that can tolerate some of these limitations.
```
Comment (jaybee84) [inline content/02.intro.md]:
```suggestion
In this perspective, we discuss techniques for understanding the nature of rare disease data, methods for addressing some of the limitations of these data, and machine learning methods that can tolerate some of these limitations. 
```

**Review by jaybee84 — APPROVED** [review]
The order looks good to me too. I think we had a similar arrangement in our abstract in the beginning, but finally re-shuffled the order to emphasize the article's focus on the prior knowledge section and ml methodology.

**Pull request merged**